src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +105,10 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates.stream()
+                .filter(new CandidateValidator())
+                .sorted(Comparator.comparing(Candidate::getName))
+                .map(Candidate::getName)
```

Comment:
```suggestion
                .map(Candidate::getName)
                .sorted()
```
I'd recommend something like this